src/tests/functional/plugin/shared/include/single_op_tests/fake_convert.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/single_op/fake_convert.hpp"
+
+namespace ov {
+namespace test {
+
+TEST_P(FakeConvertLayerTest, Inference) {
```

Comment:
Is it instantiated somewhere, so we can check it passing?

Reply:
you can find instantiation for example by path openvino/src/plugins/[intel_cpu/intel_gpu/other plugin name...]/tests/functional/shared_tests_instances/single_layer_tests/